nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -213,28 +216,42 @@ def _statistic_collector_builder(collector_name: str,
     @staticmethod
     def _create_quantizer(quantizer_config: QuantizerConfig,
                           scale_shape: Tuple,
-                          statistics: MinMaxTensorStatistic) -> BaseQuantizer:
-        quantizer_cls = QUANTIZATION_MODULES.get(quantizer_config.mode)
+                          parameters: FakeQuantizeParameters,
+                          target_type: TargetType) -> BaseQuantizer:
+        mode = quantizer_config.mode
+        quantizer_cls = QUANTIZATION_MODULES.get(mode)
+        narrow_range = target_type == TargetType.OPERATION_WITH_WEIGHTS and mode == QuantizationMode.SYMMETRIC
         quantizer_spec = PTQuantizerSpec.from_config(quantizer_config,
-                                                     narrow_range=False,
+                                                     narrow_range=narrow_range,
                                                      scale_shape=scale_shape,
                                                      half_range=False,
                                                      logarithm_scale=False,
                                                      is_quantized_on_export=False,
                                                      compression_lr_multiplier=None)
         quantizer = quantizer_cls(quantizer_spec)
+
         # Fill it with minmax
-        quantizer.apply_minmax_init(min_values=statistics.min_values,
-                                    max_values=statistics.max_values)
+        PTMinMaxAlgoBackend._fill_quantizer_parameters(quantizer, parameters)
         return quantizer
 
+    @staticmethod
+    def _fill_quantizer_parameters(quantizer: BaseQuantizer, parameters: FakeQuantizeParameters) -> None:
+        quantizer.eps = 0
```

Comment:
To clarify:  scale and input range are guaranteed to be safe? > eps?

Reply:
Yes, it is guaranteed in https://github.com/openvinotoolkit/nncf/blob/eeb14c74763855e38e960476d7476194fff145dc/nncf/quantization/fake_quantize.py#L45 and https://github.com/openvinotoolkit/nncf/blob/eeb14c74763855e38e960476d7476194fff145dc/nncf/quantization/fake_quantize.py#L58

Reply:
Ok, should we than remove this check from here? https://github.com/openvinotoolkit/nncf/pull/1551/files#diff-ff4045001d7a219f6560b8f5bb4deff45c099754c8cc49faca806d075404bd9eR103